app/client/components/main.tsx

```diff
@@ -1,36 +1,44 @@
 import React from "react";
+import { initGA } from "../analytics";
 import palette from "../colours";
 import Footer from "./footer";
 import Header from "./header";
 
-export const Main: React.SFC<{}> = ({ children }) => {
```

Comment:
We should take a look at reducing the amount of SFCs at this high level in the tree, as they'll *always* re-render when their children change, even if they don't need to. `PureComponent` is more performant. SFCs should be saved for leaf nodes.

Reply:
good suggestion - feel free to change in your responsiveness PR